substrate/client/rpc-spec-v2/src/chain_head/chain_head_follow.rs

```diff
@@ -53,6 +53,10 @@ use std::{
 /// `Initialized` event.
 const MAX_FINALIZED_BLOCKS: usize = 16;
 
+/// Maximum amount of events buffered by submit_events
+/// before dropping the stream.
+const MAX_BUFFERED_EVENTS: usize = 4;
```

Comment:
Previously we used much bigger limit for [this](https://github.com/paritytech/polkadot-sdk/blob/v1.15.3-rc1/substrate/client/rpc-spec-v2/src/chain_head/subscription/inner.rs#L33) and @lexnv suggested us to make 256/512 because the number of chainHead subscriptions per connection is limited to four.

I would just make it 512 to start with.

/cc @lexnv what do you reckon?

Reply:
set to 512